recipes/poliastro/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip install --no-deps .
```

Comment:
Yes, let's use `setup.py`. Sometimes `conda` can be a bit finicky with using `pip` in a recipe. Normally we tack on a few arguments that tend to help anyways. In particular we do the following.

    python setup.py install --single-version-externally-managed --record record.txt